linux_os/guide/services/radius/group.yml

```diff
@@ -0,0 +1,11 @@
+documentation_complete: true
+
+title: 'Radius'
```

Comment:
```suggestion
title: 'Remote Authentication Dial-In User Service (RADIUS)'
```

Reply:
Small nit to expand group title, since this will likely be the first occurrence of the term.

Reply:
Will do.